packages/api/cms-api/schema.gql

```diff
@@ -16,6 +16,20 @@ enum UserPermissionSource {
   BY_RULE
 }
 
+enum CombinedPermission {
```

Comment:
I know it is a POC but maybe we could find a better name 
- Permissions
- AllPermissions

Reply:
Difficult: `Permission` would be correct, but we already have a `Permission` type. Maybe we can name it just in the Gql schema "Permission" and keep the enum CombinedPermission name in api code.

Reply:
It's possible to export/import both a variable and a type with the same name: 9ced81585ca46175895f20bab2987b0fced7c467. This may be confusing though. Do we want this? Otherwise I can revert the change and only rename the GraphQL enum.

Reply:
> It's possible to export/import both a variable and a type with the same name: https://github.com/vivid-planet/comet/commit/9ced81585ca46175895f20bab2987b0fced7c467. This may be confusing though. Do we want this? Otherwise I can revert the change and only rename the GraphQL enum.

I think this is much better 👍

Reply:
> Usually the variable's should be of the exported type with the same name. (even if this is only a convention)

Should we revert it? Or rename to PermissionType and PermissionEnum?

Reply:
I reverted the rename of the variable and changed the name to "Permission" in the GraphQL schema only: 7260f7c65d611ed528b0d35ea842a5da85e2772e. IMO "CombinedPermission" is a fitting name since it indicates that something is combined (core and application permissions), but I'm open to "AllPermission" as well.